gavel.py

```diff
@@ -304,62 +108,62 @@ def admin():
     return render_template('admin.html', annotators=annotators, counts=counts, item_counts=item_counts, items=items, votes=len(decisions))
 
 @app.route('/admin/item', methods=['POST'])
-@requires_auth
+@utils.requires_auth
 def item():
     action = request.form['action']
     if action == 'Submit':
         csv = request.form['data']
-        data = data_from_csv_string(csv.encode('utf8'))
+        data = utils.data_from_csv_string(csv.encode('utf8'))
         for row in data:
-            item = Item(*row)
-            db.session.add(item)
+            _item = Item(*row)
+            db.session.add(_item)
         db.session.commit()
     elif action == 'Delete':
-        id = request.form['item_id']
+        item_id = request.form['item_id']
         try:
-            db.session.execute(ignore_table.delete(ignore_table.c.item_id == id))
-            Item.query.filter_by(id=id).delete()
+            db.session.execute(ignore_table.delete(ignore_table.c.item_id == item_id))
+            Item.query.filter_by(id=item_id).delete()
             db.session.commit()
         except IntegrityError as e:
             return render_template('error.html', message=str(e))
     return redirect(url_for('admin'))
 
 @app.route('/admin/annotator', methods=['POST'])
-@requires_auth
-def annotator():
+@utils.requires_auth
+def create_annotator():
```

Comment:
Right, but having a descriptive name is good so we don't confuse other people working on the code.

(yeah, this is a kinda badly designed endpoint that does a whole bunch of stuff, so maybe it could be redesigned eventually)